### PR TITLE
Fixes for v14

### PR DESCRIPTION
**Pull request by bdunne**

### CODE REVIEW

**Comment by Fryguy** [comment]
We should include the script changes here as well so we don't miss them in the future.  I know I had that Source1 one, but I guess it didn't work

**Comment by Fryguy** [comment]
hmmm - https://github.com/ManageIQ/manageiq-rpm_build/blob/master/lib/tasks/release.rake#L174

**Comment by bdunne** [comment]
> We should include the script changes here as well so we don't miss them in the future. I know I had that Source1 one, but I guess it didn't work

Done

File: lib/tasks/release.rake [hunk 1]
```diff
@@ -149,9 +149,11 @@ namespace :release do
     content = options.read
     content.sub!(/(rpm:\n\s+version:\s+).+/, "\\1#{next_branch_number}.0.0")
     content.sub!(/^(\s{2}release:\s+).+$/, "\\10")
-    content.sub!(/^(\s{4}:)\d+\w+(:)/, "\\1#{rpm_repo_name}\\2")
+    content.sub!(/^(\s{4}:)\d+-\w+(:)/, "\\1#{rpm_repo_name}\\2")
+    content.sub!(/^(\s{4}:)\d+-\w+(-nightly:)/, "\\1#{rpm_repo_name}\\2")
```
[the remark this answers is not in the document]
Comment (Fryguy) [reply lib/tasks/release.rake:153]: Actually no, scratch that - I didn't realize you wanted to "preserve" the nightly portion.

File: lib/tasks/release.rake [hunk 2]
```diff
@@ -149,9 +149,11 @@ namespace :release do
...
     content.sub!(/^(\s{8}:manageiq:[^\d]+)\d+/, "\\1#{next_branch_number}")
     content.sub!(/^(\s{8}:manageiq-release:[^\d]+)\d+/, "\\1#{next_branch_number}")
+    content.sub!(/^(\s{8}:manageiq-nightly:[^\d]+)\d+/, "\\1#{next_branch_number}")
```
Comment (Fryguy) [inline lib/tasks/release.rake:156]: Can this be simplified to:

```suggestion
    content.sub!(/^(\s{8}:manageiq[^:]*:[^\d]+)\d+/, "\\1#{next_branch_number}")
```

**Comment by Fryguy** [comment]
I'm gonna merge as is, and we can cleanup anything after.

**Pull request merged**